cypress/tests/ImageElement.spec.ts

```diff
@@ -224,13 +238,27 @@ describe("ImageElement", () => {
         setDocFromHtml(values);
         assertDocHtml(getSerialisedHtml(values));
       });
+
+      it.only("should select all of its text when the select all shortcut is used", () => {
```

Comment:
Don't forget to strip the `.only`s! Could have a CI linting rule, I do this all the time 😅

Reply:
Whoops